components/mpas-ocean/src/Registry.xml

```diff
@@ -3792,10 +3788,6 @@
 			description="Latitude function for tidal constituents: long-period = 3\sin^2(\phi)-1, diurnal = \sin(2\phi), semi-diurnal = \cos^2(\phi)"
 			packages="tidalPotentialForcingPKG"
 		/>
-		<var name="tidalPotentialZMid" type="real" dimensions="nVertLevels nCells Time" units="m"
```

Comment:
@sbrus89 please verify that I can remove `tidalPotentialZMid`. It appears to be a remnant of something during development.